src/without.js

```diff
@@ -0,0 +1,25 @@
+var _contains = require('./internal/_contains');
+var flip = require('./flip');
+var identity = require('./identity');
+var reject = require('./reject');
+var useWith = require('./useWith');
+
+/**
+ * Returns a new list without values in the first argument.
+ * `R.equals` is used to determine equality.
+ *
+ * Acts as a transducer if a transformer is given in list position.
+ * @see R.transduce
+ *
+ * @func
+ * @memberOf R
+ * @category List
+ * @sig [a] -> [a] -> [a]
+ * @param {Array} list1 The values to be removed from `list2`.
+ * @param {Array} list2 The array to remove values from.
+ * @return {Array} The new array without values in `list1`.
+ * @example
+ *
+ *      R.without([1, 2], [1, 2, 1, 3, 4]); //=> [3, 4]
+ */
+module.exports = useWith(reject, [flip(_contains), identity]);
```

Comment:
I'd prefer not to use `useWith` internally. A lambda would reduce the number of Ramda functions involved in this definition, reducing the footprint of `require('ramda/src/without')` when using Browserify or similar.